style/sass/sample.scss

```diff
@@ -1,53 +1,35 @@
-@import "bourbon";
+@import "partial-name";
```

Comment:
What about `@import "stylesheet"`. My gut reaction to seeing "partial-name" is that its too programatic. Don't care that much though.